api/types.proto

```diff
@@ -862,6 +862,9 @@ message Placement {
 	// This field is used in the platform filter for scheduling. If empty,
 	// then the platform filter is off, meaning there are no scheduling restrictions.
 	repeated Platform platforms = 3;
+
+	// Max replicas specifies limit for maximum number of replicas running on one node.
+	uint64 maxreplicas = 4;
```

Comment:
nit: calling this `max_replicas` should get it to compile to go as `MaxReplicas`

Additionally, the comment should then be of the form

```
// MaxReplicas specifies the limit . . . 
```